{"variants":[{"paths":["\/documentation\/dotfacecore\/cameraimagesource\/error\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"identifier":{"url":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/LocalizedError-Implementations","interfaceLanguage":"swift"},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/errorDescription-628h9","doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/failureReason","doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/helpAnchor","doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/recoverySuggestion"],"generated":true}],"kind":"article","metadata":{"modules":[{"name":"DotFaceCore"}],"role":"collectionGroup","title":"LocalizedError Implementations"},"hierarchy":{"paths":[["doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore","doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource","doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error"]]},"references":{"doc://com.innovatrics.dot.face.core/documentation/DotFaceCore/CameraImageSource/Error/recoverySuggestion":{"role":"symbol","title":"recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/recoverySuggestion","kind":"symbol","type":"topic","url":"\/documentation\/dotfacecore\/cameraimagesource\/error\/recoverysuggestion"},"doc://com.innovatrics.dot.face.core/documentation/DotFaceCore/CameraImageSource/Error":{"role":"symbol","title":"CameraImageSource.Error","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Error"}],"abstract":[],"identifier":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Error"}],"url":"\/documentation\/dotfacecore\/cameraimagesource\/error"},"doc://com.innovatrics.dot.face.core/documentation/DotFaceCore/CameraImageSource/Error/failureReason":{"role":"symbol","title":"failureReason","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/failureReason","kind":"symbol","type":"topic","url":"\/documentation\/dotfacecore\/cameraimagesource\/error\/failurereason"},"doc://com.innovatrics.dot.face.core/documentation/DotFaceCore/CameraImageSource/Error/helpAnchor":{"role":"symbol","title":"helpAnchor","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/helpAnchor","kind":"symbol","type":"topic","url":"\/documentation\/dotfacecore\/cameraimagesource\/error\/helpanchor"},"doc://com.innovatrics.dot.face.core/documentation/DotFaceCore/CameraImageSource":{"role":"symbol","title":"CameraImageSource","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"CameraImageSource"}],"abstract":[],"identifier":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"CameraImageSource"}],"url":"\/documentation\/dotfacecore\/cameraimagesource"},"doc://com.innovatrics.dot.face.core/documentation/DotFaceCore":{"role":"collection","title":"DotFaceCore","abstract":[{"type":"text","text":"version: 5.1.1"}],"identifier":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore","kind":"symbol","type":"topic","url":"\/documentation\/dotfacecore"},"doc://com.innovatrics.dot.face.core/documentation/DotFaceCore/CameraImageSource/Error/errorDescription-628h9":{"role":"symbol","title":"errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/com.innovatrics.dot.face.core\/documentation\/DotFaceCore\/CameraImageSource\/Error\/errorDescription-628h9","kind":"symbol","type":"topic","url":"\/documentation\/dotfacecore\/cameraimagesource\/error\/errordescription-628h9"}},"variantOverrides":[{"patch":[{"op":"replace","path":"\/references\/doc:~1~1com.innovatrics.dot.face.core~1documentation~1DotFaceCore~1CameraImageSource\/title","value":"CameraImageSource"},{"op":"replace","path":"\/references\/doc:~1~1com.innovatrics.dot.face.core~1documentation~1DotFaceCore~1CameraImageSource\/fragments","value":[{"kind":"identifier","text":"CameraImageSource"}]}],"traits":[{"interfaceLanguage":"occ"}]}]}